src/__tests__/patients/care-goals/CareGoalForm.test.tsx

```diff
@@ -253,41 +247,41 @@ describe('Care Goal Form', () => {
-    const achievementStatusSelector = wrapper.findWhere(
-      (w) => w.prop('name') === 'achievementStatus',
+    const alert = await screen.findByRole('alert')
+    const descriptionInput = screen.getByRole('textbox', {
+      name: /this is a required input/i,
+    })
+    const priority = within(screen.getByTestId('prioritySelect')).getByRole('combobox')
+    const status = within(screen.getByTestId('statusSelect')).getByRole('combobox')
+    const achievementStatus = within(screen.getByTestId('achievementStatusSelect')).getByRole(
+      'combobox',
     )
-    const startDatePicker = wrapper.findWhere((w) => w.prop('name') === 'startDate')
-    const dueDatePicker = wrapper.findWhere((w) => w.prop('name') === 'dueDate')
-    const noteInput = wrapper.findWhere((w) => w.prop('name') === 'note')
-
-    expect(alert).toHaveLength(1)
-    expect(alert.prop('message')).toEqual(expectedError.message)
-
-    expect(descriptionInput.prop('isInvalid')).toBeTruthy()
-    expect(descriptionInput.prop('feedback')).toEqual(expectedError.description)
-
-    expect(prioritySelector.prop('isInvalid')).toBeTruthy()
-    // expect(prioritySelector.prop('feedback')).toEqual(expectedError.priority)
-
-    expect(statusSelector.prop('isInvalid')).toBeTruthy()
-    // expect(statusSelector.prop('feedback')).toEqual(expectedError.status)
-
-    expect(achievementStatusSelector.prop('isInvalid')).toBeTruthy()
-    // expect(achievementStatusSelector.prop('feedback')).toEqual(expectedError.achievementStatus)
-
-    expect(startDatePicker.prop('isInvalid')).toBeTruthy()
-    expect(startDatePicker.prop('feedback')).toEqual(expectedError.startDate)
-
-    expect(dueDatePicker.prop('isInvalid')).toBeTruthy()
-    expect(dueDatePicker.prop('feedback')).toEqual(expectedError.dueDate)
+    const startDatePicker = within(screen.getByTestId('startDateDatePicker')).getByRole('textbox')
+    const dueDatePicker = within(screen.getByTestId('dueDateDatePicker')).getByRole('textbox')
+    const noteInput = screen.getByRole('textbox', {
+      name: /patient\.caregoal\.note/i,
+    })
 
-    expect(noteInput.prop('isInvalid')).toBeTruthy()
-    expect(noteInput.prop('feedback')).toEqual(expectedError.note)
+    expect(screen.getByText(/some description error/i)).toBeInTheDocument()
+    expect(screen.getByText(/some start date error/i)).toBeInTheDocument()
+    expect(screen.getByText(/some due date error/i)).toBeInTheDocument()
+
+    expect(alert).toHaveTextContent(expectedError.message)
+    expect(descriptionInput).toBeInTheDocument()
+    expect(priority).toBeInTheDocument()
+    expect(status).toBeInTheDocument()
+    expect(achievementStatus).toBeInTheDocument()
+    expect(startDatePicker).toBeInTheDocument()
+    expect(dueDatePicker).toBeInTheDocument()
+    expect(noteInput).toBeInTheDocument()
+
+    // TODO: not using built-in form accessibility features yet: HTMLInputElement.setCustomValidity()
```

Comment:
Just adding this here so we dont forget the TODO

Reply:
@nobrayner @codyarose  this might be a good start for issues